ozark/pom.xml

```diff
@@ -7,12 +7,53 @@
         <artifactId>parent</artifactId>
         <version>1.0.0-m03-SNAPSHOT</version>
     </parent>
-    <artifactId>ozark</artifactId>
-    <packaging>jar</packaging>
-    <name>Ozark ${project.version} MVC RI</name>
+    <artifactId>javax.mvc</artifactId>
```

Comment:
I don't think there's a convention at all. Mojarra uses `com.sun.faces:jsf-impl` and Jersey `org.glassfish.jersey.core:jersey-server`. The GAV [has already been changed](https://github.com/mvc-spec/ozark/commit/9c21a84f616e971dcd809d5a1c830bdf9a006dce) with [OZARK-51](https://java.net/jira/browse/OZARK-51) so I wouldn't change it again. Maybe you could start a discussion on the EG list?